packages/stopwatch/tsup.config.ts

```diff
@@ -1,3 +1,26 @@
 import { createTsupConfig } from '../../scripts/tsup.config';
 
-export default createTsupConfig({ format: ['esm', 'cjs'] });
+export default createTsupConfig({
+	format: ['esm', 'cjs', 'iife'],
+	globalName: 'Stopwatch',
+	esbuildOptions: (options, context) => {
+		switch (context.format) {
+			case 'cjs': {
+				options.banner = {
+					js: 'const { performance } = require("node:perf_hooks");'
+				};
+				break;
+			}
+			case 'esm': {
+				options.banner = {
+					js: "import { performance } from 'node:perf_hooks';"
+				};
+				break;
+			}
+			case 'iife': {
+				// iife doesn't need any fetch replacements
+				break;
+			}
+		}
+	}
```

Comment:
CJS and ESM don't need imports either, `performance` is available globally in Node:

![image](https://user-images.githubusercontent.com/4019718/193356871-925f95f7-e609-4638-8dc8-b0e1f38030fe.png)